erikiado/jp2_online
# Task/delete estudio socioeconomico

#### What's this PR do?
It allows a capturista to soft delete a estudio socioeconómico, from his dashboard
#### Where should the reviewer start?
In captura/views.py estudio_delete_modal, estudio_delete
Afterthat check the forms, as well as the modified templates
This work follows a similar structure to how erikiado implemented the delition of users.

#### How should this be manually tested?
Login as capturista, create a new study, add two inegrantes one of which should be a student
Then return to your dashboard and delete the study with the button, you'll be prompted to confirm the delition.

#### Any background context you want to provide?



This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
